src/eventshub/src/component/SortBy/SortBy.tsx

```diff
@@ -41,10 +42,10 @@ export const SortBy = ({ activeTab }: SortByProps) => {
   return (
     <div className="er-flex er-direction-row er-items-center er-w-fit er-ml-auto">
       <label htmlFor={id} className="er-m-0 er-max-w-fit er-w-full er-text-16 er-h-fit">Sort by: </label>
-      <select id={id} onChange={handleChange} value={activeSort} className="er-m-0 er-py-0 er-pl-6 er-pr-36 er-max-w-fit er-w-full er-text-16">
-        <option value='relevance'>Relevance</option>
-        <option value='asc'>Name: A-Z</option>
-        <option value='desc'>Name: Z-A</option>
+      <select id={id} value={activeSort} onChange={handleChange} className="er-m-0 er-py-0 er-pl-6 er-pr-36 er-max-w-fit er-w-full er-text-16">
+        <option aria-selected={(activeSort === 'relevance') ? true : undefined} value='relevance'>Relevance</option>
+        <option aria-selected={(activeSort === 'asc') ? true : undefined} value='asc'>Name: A-Z</option>
```

Comment:
I punted and added aria-selected here. Ugly yes, and I have to check in to see if CJWest thinks this is valid or if I should go flog myself for trying something so dastardly.

Reply:
Haha! Please don't flog yourself, @sherakama!
I don't know this code enough to see what dangers lurk. Let me know where I can look at this when available. 🍀

Reply:
Went with it! Thanks.